api/routes/period.js

```diff
@@ -0,0 +1,60 @@
+var express = require("express")
+var router = express.Router();
+var jwt = require('jsonwebtoken');
+var crypto = require('crypto');
+var morgan  = require('morgan');
+const mongoose = require('mongoose');
+const period = require('../models/period')
+
+//Set up default mongoose connection
+module.exports = router;
+
+router.get("/students/update", function(req, res) {
+    period.findByIdAndUpdate(req.body.period_id, req.body.period, function(err, result) {
+      if (err) {
+        res.send(err);
+      } else {
+        res.send(result);
+      }
+    });
+});
+
+//Update Query
+router.get("/students/update/:period_id", function(req, res) {
+    period.findByIdAndUpdate(req.params['period_id'], req.body, function(err, result) {
+      if (err) {
+        res.send(err);
+      } else {
+        res.send(result);
+      }
+    });
+});
+
+router.get("/students/create", function(req,res) {
+  const per = new period({
```

Comment:
Soooo the endpoint leads me to believe that we are creating a new student inside of an existing period when we send the request, but this looks like we're creating a whole new period with one new student. Am I reading it wrong?